exchange/utils_test.go

```diff
@@ -610,93 +610,97 @@ func TestCreateSanitizedImpExt(t *testing.T) {
 		{
 			description: "imp.ext",
 			givenImpExt: map[string]json.RawMessage{
-				"anyBidder": json.RawMessage(`"anyBidderValues"`),
-				"data":      json.RawMessage(`"anyData"`),
-				"context":   json.RawMessage(`"anyContext"`),
-				"skadn":     json.RawMessage(`"anySKAdNetwork"`),
-				"gpid":      json.RawMessage(`"anyGPID"`),
-				"tid":       json.RawMessage(`"anyTID"`),
+				"unknownBidder": json.RawMessage(`"anyUnknownBidderValues"`),
```

Comment:
Nitpick: `unknownBidder` doesn't seem right for these test cases, as it's now interpreted as just an unknown value and is not checked for as a bidder.